common/src/notifications.rs

```diff
@@ -23,6 +70,34 @@ pub fn push_notification(
     });
 }
 
+#[allow(non_snake_case)]
+pub fn push_notification_actionable(
```

Comment:
the name `push_notification_actionable` may be better as `push_desktop_notification`. That is the difference - desktop notifications are being created here, while `push_notification` results in a toast being displayed only within Uplink.

Reply:
the notification for uplink only isnt generated via `push_notification`. that is handled somewhere else. `push_notification` was used for desktop notification but without action handling

Reply:
ok, my mistake. In that case, it may be good to just replace the function with the one you made and add an action that's a no-op in case anyone wants to send a notification that does nothing when you click it. 

 I still think we could pick a better name but that isn't so important. maybe something containing the words `desktop` and `notification`, and optionally a verb :grin: